CONTRIBUTING.md

```diff
@@ -119,6 +119,33 @@ Your contribution needs to meet the following standards:
   }
   ```
 
+- Similarly, avoid using `Option::unwrap`/`Result::unwrap`. Prefer propagating
```

Comment:
nit: I would remove "similarly", it does not scale If tomorrow we will need to add another condition which is similar but need another bullet point. We can simply consider this a list of rules. Start with "Avoid ..."

Reply:
Thanks! Updated :)